libvcx/src/v3/utils/encryption_envelope.rs

```diff
@@ -62,20 +64,24 @@ impl EncryptionEnvelope {
     }
 
     pub fn open(payload: Vec<u8>) -> VcxResult<A2AMessage> {
-        let unpacked_msg = crypto::unpack_message(&payload)?;
+        trace!("EncryptionEnvelope::open >>> payload: {:?}", payload);
```

Comment:
Forgotten log message? Will probably print just a byte array, no?

Reply:
Can you please be more specific?